week-3/C-array-every/exercise.js

```diff
@@ -5,7 +5,7 @@
 var students = ["Omar", "Austine", "Dany", "Swathi", "Lesley", "Rukmini"];
 var group = ["Austine", "Dany", "Swathi", "Daniel"];
 
-var groupIsOnlyStudents; // complete this statement
+var groupIsOnlyStudents = students.every(groupIsOnlyStudents); // complete this statement
```

Comment:
groupIsOnlyStudents is not defined. You need to provide a function to find the students.